src/main/resources/assets/pipez/lang/de_de.json

```diff
@@ -0,0 +1,85 @@
+{
+  "itemGroup.pipez": "Röhrez",
+  "block.pipez.item_pipe": "Item Röhre",
+  "block.pipez.fluid_pipe": "Flüssigkeits Röhre",
+  "block.pipez.energy_pipe": "Energie Röhre",
+  "block.pipez.universal_pipe": "Universele Röhre",
+  "block.pipez.gas_pipe": "Gas Röhre",
+  "item.pipez.basic_upgrade": "Normale Röhre Verbesserung",
```

Comment:
That's not proper german

Reply:
energie = Strom Rohr
Universelles = Rohr 
etc?

Reply:
I'm talking about the upgrade names

Reply:
changed to "modifikation" should fit better